var/spack/repos/builtin/packages/pegtl/package.py

```diff
@@ -19,13 +19,17 @@ class Pegtl(CMakePackage):
     git      = "https://github.com/taocpp/PEGTL.git"
 
     version('master', branch='master')
+    version('3.2.0', sha256='91aa6529ef9e6b57368e7b5b1f04a3bd26a39419d30e35a3c5c66ef073926b56')
     version('2.8.3', sha256='370afd0fbe6d73c448a33c10fbe4a7254f92077f5a217317d0a32a9231293015')
     version('2.1.4', sha256='d990dccc07b4d9ba548326d11c5c5e34fa88b34fe113cb5377da03dda29f23f2')
     version('2.0.0', sha256='5aae0505077e051cae4d855c38049cc6cf71103a6cc8d0ddef01a576e8a60cc0')
 
     # Ref: https://github.com/taocpp/PEGTL/blob/master/src/example/pegtl/json_classes.hpp
     patch('change_to_virtual_destructor.patch', when='@:2.4')
 
+    # Ref: https://bugs.gentoo.org/733678
+    patch('pegtl-2.8.3-gcc-10.patch', level=0, when='@2.1.4:2.8.3')
```

Comment:
Would it be possible to remove the patch from the repository and use a url + sha256 here?

Reply:
It would. Is anything wrong with the patch?

Reply:
Lately we recommend to host the patch out of the repo, if possible. If it's easy to add a url + sha of the patch that is preferable. You can look, iirc at boost for examples of patches that are retrieved from https.